torch/onnx/symbolic_helper.py

```diff
@@ -700,19 +700,23 @@ def _avgpool_helper(tuple_fn, padding, kernel_size, stride, divisor_override, na
     padding = tuple(tuple_fn(padding))
     return padding
 
-def assert_training_mode(op_mode, op_name):
+
+def check_training_mode(op_train_mode, op_name):
     global _training_mode
-    op_mode = True if op_mode == 1 else False
-    if op_mode != _training_mode:
-        op_mode = "training " if op_mode else "inference"
+    op_train_mode = True if op_train_mode == 1 else False
```

Comment:
Checking my understanding, how is the op_mode created for operators in the first place? 
I assume users could set modes for ops explicitly. And without it the op will take the model mode. Is that correct?

Reply:
Yes, see example in: test_batchnorm_eval_mode_train_layer